src/components/Health/Helper.ts

```diff
@@ -1,8 +1,9 @@
 import * as React from 'react';
 import { Status } from 'types/Health';
+import { Validation } from '../ConfigValidation/ConfigIndicator';
 
 type Size = 'sm' | 'md' | 'lg' | 'xl';
 
-export const createIcon = (status: Status, size?: Size) => {
+export const createIcon = (status: Status | Validation, size?: Size) => {
```

Comment:
This seems like a good oportunity to extract those properties in a Interface. Like `InterfaceIcon` or something like that, given that both `Status` and `Validation` have many equal properties.

Reply:
I was talking with joel about configindicator and health indicator they are so similar and we can extract both...but this is not the target of this PR and we have 900+ now maybe we should open a JIRA for this.

Reply:
+1 for another JIRA, this is already quite big. @aljesusg can you create one?

Reply:
Roger

Reply:
https://issues.jboss.org/browse/KIALI-3179